gradle.properties

```diff
@@ -15,3 +15,4 @@
 #
 
 org.curioswitch.curiostack.version = 0.0.143
+apiVersion = 0.0.9-SNAPSHOT
```

Comment:
Does this need to be `apiVersion` and not `version`? I don't think the java will publish correctly without a property called `version`